spec/classes/foreman_proxy__config__spec.rb

```diff
@@ -937,6 +937,19 @@
           ]
         end
 
+        context 'when use_sudoers => false' do
+          let :pre_condition do
+            'class {"foreman_proxy":
+              use_sudoers  => false,
+              use_sudoersd => false,
+            }'
+          end
+
+          it "should not modify #{etc_dir}/sudoers" do
+            should_not contain_file("#{etc_dir}/sudoers")
```

Comment:
I think this is an Augeas resource, not a file. The example still passes with the proposed change reverted.

It should probably be `should_not contain_augeas('sudo-foreman-proxy')`.